Examples/SlackClone/ChatClient.swift

```diff
@@ -27,7 +27,8 @@ extension ChatClient {
         Appearance.default = appearance
         Components.default = components
         
-        let config = ChatClientConfig(apiKey: APIKey("q95x9hkbyd6p"))
+        var config = ChatClientConfig(apiKey: APIKey("q95x9hkbyd6p"))
+        config.isLocalStorageEnabled = true
```

Comment:
@nuno-vieira @evsaev is this left over from CoreData related PR?

Reply:
This is fine. We need an example app with this enabled so that we can easily observe these issues 👍 Ideally in the future, we want to perform UI Tests on this example app to make sure it works on iOS 12. But Github Actions is limiting use on doing this for now.

Reply:
Ok. If this is intentional, this comment can be resolved.